android/app/google-services.json.example

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Do we need to include a `.example` file? Or, could we just instruct you to download the file from 1password? 

I like `.example` files when you should run `cp .env.sample .env` to create a copy and then you modify the non-sample copy. But in this scenario, you wont' be doing that - but downloading a file.

Reply:
I copied this idea from [Remote Habits](https://github.com/customerio/RemoteHabits-Android/blob/develop/app/google-services.json.example) because I think it can be helpful for anyone just trying to run the project quickly for testing, as the app won't compile without it. Particularly for customers looking for sample app, it can be helpful. However, for anyone from Customer.io, we can add instructions to download. But may be we should simply ask to download from Firebase instead of 1Password I think? As it is more convenient and easier to manage.